src/app/users/users-routing.module.ts

```diff
@@ -37,6 +39,19 @@ const routes: Routes = [
           resolve: {
             usersTemplate: UsersTemplateResolver
           }
+        },
+        {
+          path: ':id',
```

Comment:
```suggestion
          path: ':id',
          data: { title: extract('View User'), routeResolveBreadcrumb: ['user', 'username'] },
```

Reply:
@abhaychawla I have done the changes, but in this case the app doesn't show the username in the route anymore.

Reply:
@OussEmaDevCode I just checked and it seems ok to me. I see the username in the breadcrumb and the id in the route which is how it's supposed to be. If I misunderstood something, please let me know.

Reply:
@OussEmaDevCode I understand the issue. It was my oversight. Move the resolver back outside as well.